### DISCUSSION TITLE
Unable to Delete Debit Card Information from Payment Method

### DISCUSSION THREAD

**Discussion by iTanveerAhmad** [community/community#147869] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Copilot

### Body

Hello GitHub Community,

I’m encountering an issue where I am unable to delete my debit card information from the Payment Method section in my GitHub account settings. Despite following the instructions provided in the settings, I am unable to find the option to remove my payment details.

Has anyone else faced this issue? If so, could you please provide guidance or a solution on how I can successfully delete my debit card information from my account?

Thank you in advance for your help!

Best regards,
iTanveerAhmad

**Comment by davevad93** [comment]
Hi @iTanveerAhmad ,

If you are unable to remove your debit card from your billing info, you should contact support by opening a ticket [here](https://support.github.com/contact/account), select "`Billing, payments, or receipts`" and explain your issue.

**Reply by iTanveerAhmad** [reply]
Thanks @davevad93. I have sent an email to GitHub Support.
